.github/workflows/tests.yml

```
@@ -12,6 +12,12 @@ jobs:
     container: ulauncher/build-image:6.6
     steps:
       - uses: actions/checkout@v5
+      - uses: actions/cache@v4
+        with:
+          path: .venv
+          key: venv-${{ job.container.image }}-${{ hashFiles('requirements.txt') }}
```

actionlint (1.7.12)

[error] property "image" is not defined in object type {id: string; network: string}

(expression)